{"name": "Invalid value for param - roomSize in the Request Body and getting Incorrect output -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [1, 1], 'patches': 3}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628184045659, "stop": 1628184045660}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628184045660, "stop": 1628184045661}, {"name": "When Set invalid request Body { \"roomSize\" : [5, 5, 5], \"coords\" : [1, 1], \"patches\" : [[2, 2]], \"instructions\" : \"NEWSNEWS\" }", "status": "passed", "start": 1628184045661, "stop": 1628184045662}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628184045662, "stop": 1628184045673}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [1, 1], 'patches': 3}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628184045673, "stop": 1628184045674}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5, 5, 5], \"coords\" : [1, 1], \"patches\" : [[2, 2]], \"instructions\" : \"NEWSNEWS\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628184045657, "stop": 1628184045676, "uuid": "35732561-f5ea-4ebf-9d26-0b4aaf620bf0", "historyId": "e6362331f1ec96a501d297122d54a72d", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for roomSize params: Invalid value for param - roomSize in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for roomSize params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}